docs/documentation/getting-started/millionaires.md

```diff
@@ -31,15 +31,48 @@ see how things work, let's put ourselves in Elon's shoes.
 First, we upload the inputs into the Carbyne Stack
 [Amphora Secret Store](https://github.com/carbynestack/amphora). The inputs are
 the billionaires' net worth in billions. Note that this obviously has to be done
-in a private way by Jeff and Elon in a real-world setting.
+in a private way by Jeff and Elon in a real-world setting, simplified here by
+logging in as individual users.
+
+The first secret will be uploaded with the identity of Jeff. To do so please
+perform the following commands and login as Jeff using the E-Mail
+<table>
+    <tr>
+        <td>E-Mail:</td>
+        <td>`jeff@carbynestack.io`</td>
+    </tr><tr>
+        <td>Password:</td>
+        <td>`86KIo6<]!/V=`</td>
+</tr></table>
+
+!!! info
+    If you have authenticated yourself to the VCPs recently, your previous
+    session might still be cached using a browser cookie.
+
+    If you are not prompted for your credentials and not logged in as the 
+    desired user, please make sure to clear recent browser cache or cookies,
+    or re-open the tabs in private mode.
 
 ```shell
+java -jar cs.jar login
 # Create a secret representing Jeff's net worth (note that we work with 
 # billion USD here)
-export JEFFS_NET_WORTH_ID=$(java -jar cs.jar amphora create-secret 177 -t billionaire=Jeff)
+export JEFFS_NET_WORTH_ID=$(java -jar cs.jar amphora create-secret 177 -t billionaire=Jeff -t accessPolicy=carbynestack.def -t authorizedPrograms=ephemeral-generic)
+```
+
+Next we will log in as Elon to perform the remaining steps of the tutorial. The
```

Comment:
Using "we" is a change of style. I would suggest to just use "Next log in as ..."

Reply:
changed